### ISSUE TITLE
`lspci` parser PhySlot issue

### ISSUE THREAD

**Issue by wwitzel3** [kellyjonbrazil/jc#554]
My system has some output from `lspci` that is as follows:

```
Slot:   38:00.0
Class:  PCI bridge [0604]
Vendor: Broadcom / LSI [1000]
Device: PEX890xx PCIe Gen 5 Switch [c030]
PhySlot:        255-2
Rev:    b0
NUMANode:       0
IOMMUGroup:     43
```

This PhySlot fails to parse with `jc` because of the attempt to cast `255-2` to an integer.

https://github.com/kellyjonbrazil/jc/blob/master/jc/parsers/lspci.py#L162

**Comment by wwitzel3** [comment]
The work around for now is to use `raw=True` to avoid going through the step of transforming the data to the types defined in the schema.

**Comment by kellyjonbrazil** [comment]
Thank you for reporting this! I didn't realize that field would be a non-integer value. Sounds like I should just keep that a string field. (no integer conversion)

**Comment by kellyjonbrazil** [comment]
This is fixed in the `dev` branch. This will make it into the next release.

https://github.com/kellyjonbrazil/jc/blob/dev/jc/parsers/lspci.py

**Comment by kellyjonbrazil** [comment]
Fixed in v1.25.3
